csvcubed/csvcubed/cli/inspect/inspect.py

```diff
@@ -0,0 +1,80 @@
+"""
+Inspect Command
+---------------
+
+Output CSV-W metadata in a user-friendly format to the CLI for validation.
+"""
+
+from asyncio.log import logger
```

Comment:
Is this used?

Reply:
This import is redundant - I have removed it.